lib/MusicBrainz/Server/Edit/Alias/Edit.pm

```diff
@@ -167,10 +169,7 @@ sub initialize
     die "You must specify the alias object to edit" unless defined $alias;
     my $entity = delete $opts{entity} or die 'Missing "entity" argument';
 
-    unless (non_empty($opts{sort_name})) {
-        delete $opts{sort_name};
-        $opts{sort_name} = $opts{name} if non_empty($opts{name});
-    }
```

Comment:
I think it's just because the tests for `Edit::Artist::EditAlias` don't pass in a name when constructing the edits. Those tests fail if I remove that part. I see the name is required by FormHandler, so perhaps this can't be triggered through the UI. I guess I'd lean toward keeping it so we don't have to update the tests, and in case we ever allow this edit type in /ws/js/edit, which I imagine would happily accept only the fields that changed.